Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift

```diff
@@ -224,4 +238,18 @@ class MSAnalyticsViewController: UITableViewController, AppCenterProtocol {
     }
     return super.tableView(tableView, cellForRowAt: indexPath)
   }
+  
+  func initLatencyPicker() {
+    let latencyPosition = UserDefaults.standard.integer(forKey: kMSTransmissionIterval)
+    self.latencyPicker = MSEnumPicker<Latency>(
+    textField: latencyField,
+    allValues: Latency.allValues,
+    onChange: { index in
+      UserDefaults.standard.setValue(index, forKey: kMSTransmissionIterval)
```

Comment:
Store value instead of the index and get rid of additional enums in AppDelegates